assets/js/blocks/cart/edit.js

```diff
@@ -38,6 +38,7 @@ import {
 	useForcedLayout,
 } from '../cart-checkout-shared';
 import { CartBlockContext } from './context';
+import { CartSidebarCompatibilityNotice } from '@woocommerce/editor-components/sidebar-compatibility-notice';
```

Comment:
We should move this new notice component to the `External dependencies` list. And remove the old one in `line 15`. 🙌